airflow/providers/google/suite/hooks/sheets.py

```diff
@@ -375,3 +375,60 @@ def batch_clear(self, spreadsheet_id: str, ranges: List) -> Dict:
         ).execute(num_retries=self.num_retries)
 
         return response
+
+    def get_sheet_titles(self, spreadsheet_id: str, sheet_filter: Optional[List[str]] = None):
+        """
+        Retrieves the sheet titles from a spreadsheet matching the given id and sheet filter.
+
+        :param spreadsheet_id: The spreadsheet id.
```

Comment:
Why there are no :type param: here and within others methods?

Reply:
We have type annotations, however, I think I should add them for docs sake - @mik-laj ?